etcdctl/README.md

```diff
@@ -51,13 +53,23 @@ The protobuf encoding of the PUT [RPC response][etcdrpc].
 #### Examples
 
 ```bash
-./etcdctl put foo bar --lease=1234abcd
+./etcdctl put foo bar --lease 1234abcd
 # OK
 ./etcdctl get foo
 # foo
 # bar
 ```
 
+```bash
+./etcdctl put foo bar1 --prev-kv --lease 1234abcd
```

Comment:
sorry i mean we should remove the flag `--lease=1234abcd` in this example since it is irrelevant in this example. not change `--lease=1234abcd` to `--lease 1234abcd`. we should change other places back too.